### DISCUSSION TITLE
Can't complete Tinacms Project Setup Checklist step 2

### DISCUSSION THREAD

**Discussion by RinFroster** [tinacms/tinacms#4166] [Help]
Hi, I'm currently new to Tinacms and want to connect it with me Nuxt 3 project but i can't seem to pass the Poject Setup Checklist step 2.
I already run npx @tinacms/cli@latest init and deploy it to github but still can't see the checklist update step 2, please help me
This is my github repo: https://github.com/RinFroster/nuxt-project
My package.json and my .env file here:
![image](https://github.com/tinacms/tinacms/assets/61495503/3f6e0c4f-0ef4-4872-b2e5-16e22a90303f)
![image](https://github.com/tinacms/tinacms/assets/61495503/f222ff3d-c45a-4315-af3e-89382f0b697a)
Appreciate any help!

**Comment by kldavis4** [comment]
Is your repo public? Getting a 404 when trying to access it.

**Reply by RinFroster** [reply]
Oh sorry my bad, i have change it to public now

**Reply by kldavis4** [reply]
thanks. I imported the project into tina and see the following:
<img width="574" alt="Screen Shot 2023-08-29 at 09 39 45" src="https://github.com/tinacms/tinacms/assets/370955/3aa963f7-2111-4114-905f-dd16a97090b9">

is that different from what you are seeing?

**Reply by RinFroster** [reply]
yes, my step 2 not active and the admin page on vercel link get page not found so i'm unable to edit anything except access it on my localhost
![image](https://github.com/tinacms/tinacms/assets/61495503/acdc6fdf-e652-4b02-a291-9fd0a0d4efc4)

**Reply by RinFroster** [reply]
when i access /admin on vercel it get page not found like this:
![image](https://github.com/tinacms/tinacms/assets/61495503/e28892a1-491d-478c-823a-4c95ab9cf886)

**Reply by kldavis4** [reply]
I think the checklist is probably just a Tina Cloud bug. The missing /admin is probably because the /admin/index.html isn't getting output to the correct location during the tinacms build.
